plugins/scaleway/scw.go

```diff
@@ -0,0 +1,27 @@
+package scaleway
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/needsauth"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+)
+
+func ScalewayCLI() schema.Executable {
+	return schema.Executable{
+		Name:    "Scaleway CLI",
+		Runs:    []string{"scw"},
+		DocsURL: sdk.URL("https://www.scaleway.com/en/cli"),
+		NeedsAuth: needsauth.IfAll(
```

Comment:
Let's skip for `-p` as well, because that applies only when the configuration file exists on the computer. So, it's not relevant when a shell plugin is being used.